hathor/util.py

```diff
@@ -482,6 +492,11 @@ def progress(iter_tx: Iterator['BaseTransaction'], *, log: Optional['structlog.s
                 log.info(f'loading... {math.floor(progress * 100):2.0f}%', progress=progress, **kwargs)
             else:
                 log.info('loading...', **kwargs)
+            # XXX: this collections will happen every _DT_LOG_PROGRESS (=30s) on average, which is good, and because
+            #      automatic collection should be disabled, it won't happen during processing of transactions, which
+            #      can make it seem like a transaction took more time to be processed when has nothing to do with the
+            #      transaction itself
+            gc.collect()
```

Comment:
Does it make sense to have another `gc.collect()` in the end of the method after the while breaks? We might break the while right before the `_DT_LOG_PROGRESS`.

It probably won't affect that much, given that after this method ends, the automatic gc will start again and clean it soon but I'm not sure.

Reply:
Yeah, it totally makes sense, at some point I thought of doing that too but then I forgot. I'll do that, thanks.

Reply:
Done.